### PR TITLE
config: Add log configuration items to the config file

### PR DESCRIPTION
**Pull request by SuperSupeng**
### What problem does this PR solve? 

Add log configuration items to the config file. #1384
### What is changed and how it works?
Add logFileMaxSize, logFileMaxDays and logFileMaxBackups items when load config file.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change
 - Has exported variable/fields change

### Release note

- No release note

### CODE REVIEW

**Comment by SuperSupeng** [comment]
ti-chi-bot https://github.com/pingcap/ticdc/pull/2194

File: pkg/config/config.go [hunk 1]
```diff
@@ -142,11 +142,30 @@ func GetDefaultReplicaConfig() *ReplicaConfig {
...
+// LogFileConfig represents log file config for server
+type LogFileConfig struct {
+	MaxSize    int `toml:"max-size" json:"max-size"`
+	MaxDays    int `toml:"max-days" json:"max-days"`
+	MaxBackups int `toml:"max-backups" json:"max-backups"`
+}
+
+// LogConfig represents log config for server
+type LogConfig struct {
+	File *LogFileConfig `toml:"file" json:"file"`
```
Comment (liuzix) [inline pkg/config/config.go:154]: Is it necessary to embed yet another struct `LogFileConfig` into `LogConfig`?

Comment (SuperSupeng) [reply pkg/config/config.go:154]: The configuration file is in this form to try to keep the TiCDC as consistent as possible with the TiDB configuration.

**Review by overvenus — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request merged**